### PR TITLE
Add Brimble / PaaS

### PR DESCRIPTION
**Pull request by klefcodes**
### website:
https://brimble.io

### examples:
https://www.sidekyk.app
https://usecrim.brimble.app
https://getfastrack.app
https://heinz.brimble.app

### CODE REVIEW

File: src/technologies/b.json [hunk 1]
```diff
@@ -2765,5 +2765,22 @@
...
+  "brimble": {
+    "cats": [
+      62
+    ],
+    "description": "Brimble is a cloud platform for deploying frontend web applications",
+    "headers": {
+      "server": "^Brimble$",
+      "x-brimble-id": ""
+    },
+    "icon": "brimble.svg",
```
Comment (nurbek91) [inline src/technologies/b.json:2778]: Brimble
32x32

File: src/technologies/b.json [hunk 2]
```diff
@@ -2765,5 +2765,22 @@
     "saas": true,
     "scriptSrc": "cdn\\.ibdok\\.ir/",
     "website": "https://bdok.ir"
+  },
+  "brimble": {
```
Comment (nurbek91) [inline src/technologies/b.json:2769]: Brimble

File: src/technologies/b.json [hunk 3]
```diff
@@ -2765,5 +2765,22 @@
...
+    "cats": [
+      62
+    ],
+    "description": "Brimble is a cloud platform for deploying frontend web applications",
```
Comment (nurbek91) [inline src/technologies/b.json:2773]: "description": "Brimble is a cloud platform for deploying frontend web applications.",

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks for PR!!!
Cosmetic changes required

**Review by nurbek91 — APPROVED** [review]

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**